src/SFA.DAS.ApprenticeCommitments/Apis/InnerApi/CreateApprenticeshipRequest.cs

```diff
@@ -17,6 +17,7 @@ public class CreateApprenticeshipRequestData
         public string Email { get; set; }
         public string EmployerName { get; set; }
         public long EmployerAccountLegalEntityId { get; set; }
+        public string TrainingProviderName { get; set; }
```

Comment:
Definitely TrainingProviderId  should be here

Reply:
Yep, I've added it in with a dummy value just like the name.